vsphere_plugin_common/__init__.py

```diff
@@ -1664,14 +1718,23 @@ def create_server(
         relospec.datastore = datastore.obj
         relospec.pool = resource_pool.obj
         if not auto_placement:
-            logger().warn(
+            self._logger.warn(
                 'Disabled autoplacement is not recomended for a cluster.'
             )
             relospec.host = host.obj
 
+        # get list nic mac's for remove
+        macs_for_remove = []
+        if postpone_delete_networks and not enable_start_vm:
+            macs_for_remove = self._get_nic_keys_for_remove(template_vm)
+
+        if postpone_delete_networks and enable_start_vm:
+            self._logger.info("Use postpone delete networks with "
```

Comment:
*Using postpone_delete_networks with enable_start_vm is unsupported.

Reply:
fixed